manim/mobject/numbers.py

```diff
@@ -139,3 +140,28 @@ class Integer(DecimalNumber):
 
     def get_value(self):
         return int(np.round(super().get_value()))
+
+
+class Variable(VMobject):
+    def __init__(
+        self, var, label, var_type=DecimalNumber, num_decimal_places=2, **kwargs
+    ):
+        # if the label contains underscores, we need to escape them
+        # because otherwise the next letter after underscore will be subscripted
+        label = label.replace("_", "\_")
+        self.label = TexMobject(label + "=")
```

Comment:
Question: Is TexMobject in math mode by default? (I always mix TexMobject and TextMobject...)

Suggestion: the label is showing up in cursive, which makes me think that you are writing LaTeX in math mode. I think it should be in text mode, no?

Reply:
> Is TexMobject in math mode by default? (I always mix TexMobject and TextMobject...)

As far as I know, `TexMobject` is for writing equations and `TextMobject` is for normal text. If there is something like a math mode and text mode, I am currently unaware about it.

> Suggestion: the label is showing up in cursive, which makes me think that you are writing LaTeX in math mode. I think it should be in text mode, no?

Yes, I used `TexMobject` so that even labels for variables with subscripts and superscripts are allowed.
Let me know if I understood correctly what you are trying to say :)

Reply:
Yeah by math mode I meant LaTeX.

By your description, it seems like TexMobject seems to be in mathmode by default. In this case, it's weird that the subscripted variables didn't work. What happened when you tried it?

Reply:
The subscripted values didn't work because I replaced all underscores with backslash-underscores, so they got escaped and I got "a_i" instead of a (sub) i.

Reply:
I just changed TexMobject to TextMobject. So for `var_name`, users have to pass the string "var_name" and for `a sub i`, users have to pass in "${ a }_{ i }$"

Reply:
It does for me, yes :)